### PR TITLE
Additional notes to README.md 3635 Corrected

### LINKED ISSUE #3635: Updating Wiki: Install MkDocs
Dependency
- #3634

### Overview
We need to install MkDocs into the new repository so that we can start to move the content from old wiki to new wiki

### Action Items
- [x] research MkDoc requirements
- [x] make a record of the research/requirements and choose a place in the wiki to keep the research 
 - [x] read the documentation on MkDocs for any additional info you might need to install (and update our documentation with the steps/choices)
 - [x] install MkDocs in the repository listed under Resources/Instructions
 - once MkDocs have been installed in the repository 
    - [x] Move issue #3636 to New Issue Approval column and remove the Dependency label
    - [x] post the link to the research/requirements under Resources/Instructions in issue #3636

### Resources/Instructions
- new repo: https://github.com/hackforla/website-wiki
- https://www.mkdocs.org/ 

#### Themes for mkdocs
- [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/getting-started/) is a theme for MkDocs, a static site generator geared towards (technical) project documentation.
- [Jekyll Doc](https://idratherbewriting.com/documentation-theme-jekyll/index.html) is a documentation theme for [Jekyll](https://jekyllrb.com/). 

#### Article comparing MK Docks vs Jekyll for WIKI
* [MkDocs or Jekyll for Technical Documentation](https://blogs.sap.com/2020/07/24/mkdocs-or-jekyll-for-technical-documentation/) - we decided to go with mkdocks

#### Examples
The Youth Justice Nav Project's documentation currently uses Material for MkDocs.
* [Youth Justice Nav repo](https://github.com/hackforla/youthjusticenav-info/tree/main/docs)
* [Youth Justice Nav documentation static site](https://hackforla.github.io/youthjusticenav-info/Juvenile%20Hall/)

GreenearthOS uses Mkdocks without material
- https://www.greenearthos.org/

#### other stuff

- [notes](https://docs.google.com/document/d/1Se4HW4Ab_UR7tgy9Ul8M0Rxev_wrL36rvPPGqYB93uU/edit)
- [audit spreadsheet](https://docs.google.com/spreadsheets/d/1QzfMX3URWfpfeoun34jRm8oDOxXMAMMRvUDS05WLpNY/edit#gid=0)
- Example of another mkdocks install https://brigade.cloud/projects/

### PR DESCRIPTION
**Pull request by t-will-gillis**
Fixes to links, additional Fixes https://github.com/hackforla/website/issues/3635
Fixes [hackforla/website-wiki/pull/2](https://github.com/hackforla/website-wiki/pull/2)

### What changes did you make and why did you make them ?
- Added the missing links noted by @JessicaLucindaCheng 
- Took the opportunity to make other edits, add info and links

### CODE REVIEW

**Comment by JessicaLucindaCheng** [comment]
- Review Availability: 2 hours
- ETA: Tue, March 21, 2023

File: README.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ The Material for MkDocs design theme includes multiple configuration and extensi
   
 
 
-### Material design customizations:  
+### Material design customizations
 
-- [Additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) - recommended placement of new style sheet 
-- [Additional JS](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) - integrating other syntax or custom logic 
-- [Theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme)
+- [Additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) - using a separate style sheet to adjust specific elements
```
Comment (JessicaLucindaCheng) [inline README.md:40]: Is this link supposed to jump to: https://squidfunk.github.io/mkdocs-material/customization/#additional-css? Currently, it goes to https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript

File: README.md [hunk 2]
```diff
@@ -85,9 +86,8 @@ The Material for MkDocs design theme includes multiple configuration and extensi
...
+  - [Tabbed](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#tabbed) - allows usage of content tabs to group related content
   - [Table of Contents](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#table-of-contents) – automatically generates a table of contents
-  - [Tables](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#tables) – adds ability of create tables in markdown  
-  - [Tabbed](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#tabbed) - allows usage of content tabs
-  - [Tasklist](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#tasklist) – allows usage of GitHub Flavored Markdown inspired tasklists  
+  - [Tables](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#tables) – adds ability of create tables in Markdown
```
Comment (JessicaLucindaCheng) [inline README.md:92]: Instead replace the `of` with `to`, so the result is `adds ability to create tables in Markdown`

File: README.md (hunk 1 above)
Comment (JessicaLucindaCheng) [inline README.md]: Is this link supposed to jump to: https://squidfunk.github.io/mkdocs-material/customization/#additional-css? If yes, please fix the link.

File: README.md (hunk 2 above)
Comment (JessicaLucindaCheng) [inline README.md]: Replace the `of` with `to`, so the result is `adds ability to create tables in Markdown`

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@t-will-gillis Thanks for making the changes requested in other pr! The additional changes you made on top of that improved the document.

I do have a few more requested changes. See my in-line comments. Thanks.

File: README.md (hunk 1 above)
Comment (t-will-gillis) [reply README.md]: Yes! it should, thank you

File: README.md (hunk 2 above)
Comment (t-will-gillis) [reply README.md]: Agreed, thank you

**Review by JessicaLucindaCheng — APPROVED** [review]
@t-will-gillis LGTM! Good job!

**Pull request merged**